src/Avalonia.Themes.Fluent/Controls/CheckBox.xaml

```diff
@@ -152,6 +152,7 @@
     <Setter Property="Data" Value="M1507 31L438 1101L-119 543L-29 453L438 919L1417 -59L1507 31Z" />
     <Setter Property="Width" Value="9" />
     <Setter Property="Opacity" Value="1" />
+    <Setter Property="FlowDirection" Value="LeftToRight" />
```

Comment:
Why do you set the FlowDirection explicitly? This disables inheritance.

Reply:
same